generated/nidaqmx/_lib.py

```diff
@@ -6,19 +6,17 @@
 import threading
 from typing import cast
 
-from nidaqmx.errors import Error
+from nidaqmx.errors import DaqNotFoundError, DaqNotSupportedError, DaqFunctionNotSupportedError
 
 
-class DaqNotFoundError(Error):
-    pass
+DAQ_NOT_FOUND_MESSAGE = "Could not find an installation of NI-DAQmx. Please ensure that NI-DAQmx " \
```

Comment:
These constants are only used in this module so I recommend making them private.